### PR TITLE
wip: example struct for user entity

### LINKED ISSUE #1: Create User.

### PR DESCRIPTION
**Pull request by igkostyuk**

### CODE REVIEW

**Comment by glbter** [comment]
@igkostyuk, I suppose, we should delete several files from day6 project structure

File: .revive.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ignoreGeneratedHeader = false
```
Comment (Metalscreame) [inline .revive.toml]: What is this file?

Comment (glbter) [reply .revive.toml]: Ihor added linter configuration file. Maybe, it's to much?
> What is this file?

**Comment by Metalscreame** [comment]
Not sure that it's fine to have structure like this `internal/user/repository/`. Having this structure you'll have to implement each layer/folder structure for each domain entity. This looks more like DDD.

I would suggest to have `internal/repository`, `internal/server/http/user.go` or `internal/server/http/user/handlers.go` and etc

**Comment by Metalscreame** [comment]
I can't see also usecase folder
`internal/usecases/user/usecase.go`

**Review by glbter — COMMENTED** [review]
As I it imagined, we would divide it into 3 independend levels, by clean arch
```
internal/
 usecases/
  /user
  ...
 repo/
  user/
  ...
 server/
  user/
  ...
```

your structure is fine, but not for monolith, it would be great for microservices

**Pull request merged**